From a67747b7a314e685085b62e8239442ea54959dbc Mon Sep 17 00:00:00 2001
From: Artem Bityutskiy <Artem.Bityutskiy@nokia.com>
Date: Fri, 15 Jan 2010 19:04:21 +0200
Subject: [PATCH 3/3] ubiformat: boost version to 1.5

We fixed the sequence numbers bug, which is quite serious.

Signed-off-by: Artem Bityutskiy <Artem.Bityutskiy@nokia.com>
---
 ubi-utils/src/ubiformat.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ubi-utils/src/ubiformat.c b/ubi-utils/src/ubiformat.c
index 2316d67..8ad5051 100644
--- a/ubi-utils/src/ubiformat.c
+++ b/ubi-utils/src/ubiformat.c
@@ -44,7 +44,7 @@
 #include "crc32.h"
 #include "common.h"
 
-#define PROGRAM_VERSION "1.4"
+#define PROGRAM_VERSION "1.5"
 #define PROGRAM_NAME    "ubiformat"
 
 /* The variables below are set by command line arguments */
-- 
1.6.5

